### PR TITLE
Lecture7

### PR DESCRIPTION
**Pull request by pliskov**
added lecture7 implementation

### CODE REVIEW

File: Lecture7/src/main/java/Helper.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        String selectedTractor = list.get(i).findElement(By.xpath("./span")).getText();
+        driver.findElements(By.xpath("//a[@class='schema-product__button button button_orange']")).get(i).click();
+        driver.findElement(By.xpath("//span[contains(.,'По цене')]/..")).click();
+        Thread.sleep(2000);
+        driver.findElements(By.xpath("//td/a[@data-href='https://cart.onliner.by']")).get(0).click();
+        return selectedTractor;
+    }
+
+    public static Boolean isInCart(String shopCartUrl, WebDriver driver, String product) throws InterruptedException {
+        Boolean exist = false;
```
Comment (acherkas) [inline Lecture7/src/main/java/Helper.java]: why you didn't use primitive? (boolean)

File: Lecture7/src/main/java/Helper.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                    "//a[@class='cart-product-title__link cart-product-title__link_name']/span"));
+            for(WebElement webElement: listCartProducts){
+                if (product.equals(webElement.getText())) {
+                    System.out.println("The product is in the cart");
+                    exist = true;
+                    return exist;
+                }
+            }
+        }
+        catch (NoSuchElementException e) {
```
Comment (acherkas) [inline Lecture7/src/main/java/Helper.java]: this exception will never appears in this case

File: Lecture7/src/main/java/Helper.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Random;
+
+public class Helper {
```
Comment (acherkas) [inline Lecture7/src/main/java/Helper.java:3]: what helper? choose correct name classes

File: Lecture7/src/test/java/AddRandomTractorToCartTest.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+public class AddRandomTractorToCartTest {
+    private final String shopUrl = "https://onliner.by";
+    private final String shopCartUrl = "https://cart.onliner.by";
+    private final Browser browser = Browser.CHROME;
+
+    @Test
+    private void addRandomTractorToCartTest() throws InterruptedException {
+        WebDriver driver = WebdriverFactory.createWebdriver(browser);
+        Assert.assertTrue(Helper.isInCart(shopCartUrl, driver, Helper.addRandomTractorToCart(shopUrl, driver)));
```
Comment (acherkas) [inline Lecture7/src/test/java/AddRandomTractorToCartTest.java]: divide for logic strings
Helper.addRandomTractorToCart(shopUrl, driver)
then assertion

File: Lecture7/src/main/java/Helper.java (hunk 1 above)
Comment (pliskov) [reply Lecture7/src/main/java/Helper.java]: accidentally. fixed

File: Lecture7/src/main/java/Helper.java (hunk 2 above)
Comment (pliskov) [reply Lecture7/src/main/java/Helper.java]: changed implementation

File: Lecture7/src/main/java/Helper.java (hunk 3 above)
Comment (pliskov) [reply Lecture7/src/main/java/Helper.java:3]: renamed to Shop

File: Lecture7/src/test/java/AddRandomTractorToCartTest.java (hunk 4 above)
Comment (pliskov) [reply Lecture7/src/test/java/AddRandomTractorToCartTest.java]: divided

**Pull request merged**